app/assets/javascripts/modules/base-target.js

```diff
@@ -0,0 +1,26 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function BaseTarget () { }
+
+  BaseTarget.prototype.start = function () {
+    if (this.pageIsInIframe()) {
+      this.setBaseTarget()
+    }
+  }
+
+  BaseTarget.prototype.setBaseTarget = function () {
+    var base = document.createElement('base')
```

Comment:
Do we know what happens if this is run twice? Is two base elements ok or should it look for one and modify it if found create if not?

Reply:
It will indeed generate two base elements - I don't see why would that happen though and by having two of the will not affect the expected behaviour. If you experienced multiple script loading in GOV.UK in the past do let me know and I'll update this script to check for an existing element and replace it if so - otherwise I would investigate the cause of having it run multiple times instead of being too protective within the code.